infra/modules/services/alertsactiongroups.bicep

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Consider merging Alert Action Group and Alerts into one file since they are very tightly coupled.